### DISCUSSION TITLE
Example Demo of TexFomula API doesn't work

### DISCUSSION THREAD

**Discussion by MikeS2058** [ForNeVeR/wpf-math#478] [Q&A]
Is there a working example showing how to use the rendering API? The one in the ReadMe doesn't work. There is no WpfTeXFormulaParser.Instance or TexFormula.RenderToPng. What do I need to do to get the example working?

**Comment by ForNeVeR** [comment]
Please clarify: how doesn't it work for you?
- [`WpfTeXFormulaParser.Instance`](https://github.com/ForNeVeR/xaml-math/blob/6f083e058f017a2f480a2ba268c4560a02c96303/src/WpfMath/Parsers/WpfTeXFormulaParser.cs#L10)
- [`Extensions.RenderToPng(this TexFormula)`](https://github.com/ForNeVeR/xaml-math/blob/6f083e058f017a2f480a2ba268c4560a02c96303/src/WpfMath/Extensions.cs#L10)

What version do you use?

**Reply by MikeS2058** [reply]
I’m making a very simple Avalonia MVVM application with just one view and view model. I want to display various math expressions in the view that are generated during runtime.

I added Avilonia-Math and Xaml-Math from Nuget to the solution according to the ReadMe instructions. 

I added the control example into the view and it works as expected.

However, when I added the API example into a class method in the same project there is no WpfMath or WpfMath.Parser name space (because WpfMath wasn’t added to the project) therefore no WpfTeXFormulaParser.Instance that is used in the example. 

I added the WpfMath nuget package but that produced a System.IO runtime error probably because there isn't any other Wpf packages added to the application.

**Reply by ForNeVeR** [reply]
Unfortunately, PNG rendering API is not yet supported for Avalonia. There's an open issue #355 about it.
